style.css

```diff
@@ -26,29 +39,147 @@
 .pagination {
     display: flex;
     justify-content: center;
+
 }
 
 .card-body{
     color: black;
+
 }
 
+.card-header-tabs{
+  margin: 10px;
+
+} 
+
 .nav-link{
     border-radius: 20px;
+
 }
+
 .selected{
     background: yellow !important;
+
+}
+
+.cover h5, .cover p{
+  width: 80%;
+
+}
+
+/* SEARCH */
+
+.search-form{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  border-radius: 6px;
+
+}
+
+
+form.search-form input[type=text] {
+  padding: 10px;
+  font-size: 17px;
+  border: 1px solid grey;
+  float: left;
+  width: 70%;
+  background: #f1f1f1;
+  border-radius: 6px;
+
+}
+
+
+form.search-form button {
+  float: left;
+  width: 50px;
+  padding: 10px;
+  margin: 5px;
+  background: #2196F3;
+  color: white;
+  font-size: 17px;
```

Comment:
use css variable `(--black-color: #000)` for colors, fonts, repetitive values

Reply:
All colors stored as variables.